common/setups/rasr/gmm_system.py

```diff
@@ -88,8 +88,12 @@ class GmmSystem(RasrSystem):
     corpus -> Path
     """
 
-    def __init__(self):
-        super().__init__()
+    def __init__(
+        self,
+        rasr_root: Optional[tk.Path] = None,
```

Comment:
```suggestion
        rasr_binary_path: Optional[tk.Path] = None,
```
We are not not passing the root, but a path to a folder directly containing the binaries, as they can be independent from the repository.